roles/gpu/tasks/nvdp.yml

```diff
@@ -20,6 +20,7 @@
   kubernetes.core.k8s:
     state: present
     src: "{{ install_dir }}/nvidia-runtime-class.yml"
+    kubeconfig: "{{ kubeconfig_path }}"
```

Comment:
Rather than specifying individual `kubeconfig` vars (and `--kubeconfig` flags) each time, would it make sense to export the `KUBECONFIG` environment variable globally once instead? It might simplify the commands and reduce repetition.